volume.go

```diff
@@ -242,3 +243,56 @@ func (c *Client) DeleteVolume(id string) (*SimpleResponse, error) {
 
 	return c.DecodeSimpleResponse(resp)
 }
+
+// GetVolumeSnapshotByVolumeID retrieves a specific volume snapshot by volume ID and snapshot ID
+func (c *Client) GetVolumeSnapshotByVolumeID(volumeID, snapshotID string) (VolumeSnapshot, error) {
+	resp, err := c.SendGetRequest(fmt.Sprintf("/v2/volumes/%s/snapshots/%s", volumeID, snapshotID))
+	if err != nil {
+		return VolumeSnapshot{}, decodeError(err)
+	}
```

Comment:
here and in all subsequent requests we need to specify region, or else even if the client is initialised with a region which is different from Default region it will still fetch resources form default region.
Ref: https://github.com/civo/civogo/blob/master/volume.go#L199

Reply:
from what I can see the post and put requests needs the region as a parameter 
for get or delete its added by default
[https://github.com/civo/civogo/blob/master/client.go#L207](https://github.com/civo/civogo/blob/master/client.go#L207)